contents/docs/api/post-only-endpoints.mdx

```diff
@@ -258,7 +258,47 @@ Body:
 }
 ```
 
-### Example request & response: /decide v2
+### Example request & response: /decide v3
+`/decide` version 3 introduces support for [feature flag payloads](TODO: update link) and ...
```

Comment:
```suggestion
`/decide` version 3 introduces support for [feature flag payloads](TODO: update link) and a change in response structure: flags that were evaluated as false are also passed in. Further, this endpoint returns a parameter `errorComputingFlags`, which is true when we didn't manage to compute some flags, because of, say the database being down or some other reason. This allows for making partial updates to currently active flags in your clients.
```